LICENSE

```diff
@@ -1,29 +1,21 @@
-BSD 3-Clause License
```

Comment:
I don't actually remember using the same license as SymPy here, but I think we should keep it.